Update the postgresql version constraint to match the version available in postgresql chart.
---

diff --git a/bitnami/mlflow/Chart.yaml b/bitnami/mlflow/Chart.yaml
index 0df240570..4a29c9d22 100644
--- a/bitnami/mlflow/Chart.yaml
+++ b/bitnami/mlflow/Chart.yaml
@@ -21,7 +21,7 @@ dependencies:
 - condition: postgresql.enabled
   name: postgresql
   repository: oci://registry-1.docker.io/bitnamicharts
-  version: 16.x.x
+  version: 17.x.x
 - name: common
   repository: oci://registry-1.docker.io/bitnamicharts
   tags:
